routes/department.js

```diff
@@ -1,11 +1,11 @@
-const express = require('express');
-const controller = require('../controllers/department');
+const express = require("express");
+const controller = require("../controllers/department");
 const router = express.Router();
 
-router.get('/', controller.getAll);
-router.get('/:id', controller.getById);
-router.delete('/:id', controller.deleteById);
-router.post('/', controller.create);
-router.patch('/:id', controller.updeteById);
+router.get("/", controller.getAll);
+router.get("/:id", controller.getById);
+router.delete("/:id", controller.deleteById);
+router.post("/", controller.create);
+router.patch("/:id", controller.updeteById);
```

Comment:
You are inconsistent with styling of the code. Setup prettier properly in the next pull request